guides/common/modules/proc_upgrading-the-external-database.adoc

```diff
@@ -1,34 +1,6 @@
 [id="Upgrading_the_External_Database_{context}"]
 = Upgrading the external database
 
-You can upgrade an external database from {EL} 8 to {EL} 9 while upgrading {Project} from {ProjectVersionPrevious} to {ProjectVersion}.
-
-.Prerequisites
-* Create a new {EL} 9 based host for PostgreSQL server that follows the external database on {EL} 9 documentation.
-ifdef::katello,orcharhino,satellite[]
-For more information, see {InstallingServerDocURL}using-external-databases_{project-context}[Using External Databases with {Project}].
-endif::[]
-* Install PostgreSQL version 13 on the new {EL} host.
-
-.Procedure
-. Create a backup.
-. Restore the backup on the new server.
-ifdef::katello,satellite,orcharhino[]
-. Correct the permissions on the `evr` extension:
-+
-[options="nowrap", subs="+quotes,verbatim,attributes"]
-----
-# runuser -l postgres -c \
-"psql -d foreman -c \"UPDATE pg_extension SET extowner = (SELECT oid FROM pg_authid WHERE rolname='foreman') WHERE extname='evr';\""
-----
-endif::[]
-. If {Project} reaches the new database server via the old name, no further changes are required.
-Otherwise reconfigure {Project} to use the new name:
-+
-[options="nowrap", subs="+quotes,verbatim,attributes"]
-----
-# {foreman-installer} \
---foreman-db-host _newpostgres.example.com_ \
---katello-candlepin-db-host _newpostgres.example.com_ \
---foreman-proxy-content-pulpcore-postgresql-host _newpostgres.example.com_
-----
+* You can upgrade an external database from PostgreSQL 12 to PostgreSQL 13 without upgrading the underlying operating system from {EL} 8 to {EL} 9. 
+* You can use the `pg_upgrade` utility to upgrade from PostgreSQL 12 to PostgreSQL 13.
```

Comment:
I'm not sure this line really adds anything because it's just repeating the line above it with more implementation details. I'd suggest to drop it.
```suggestion
```